### PR TITLE
chore: add instruction for migrate JupyterLab Classic to Current

### PR DESCRIPTION
**Pull request by Ericc-Hao**

### CODE REVIEW

File: docs/labs/jupyterlab-filelibrary.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Acess File Library within Author Workbench
```
Comment (ElioDiNino) [inline docs/labs/jupyterlab-filelibrary.md:1]:
```suggestion
# Access File Library within Author Workbench
```

File: docs/labs/jupyterlab-filelibrary.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Follow these steps to open file library in Author Workbench
+
+1. **Open Your Lab:** Begin by launching your lab detail page.
```
Comment (ElioDiNino) [inline docs/labs/jupyterlab-filelibrary.md:5]:
```suggestion
1. **Open Your Lab:** Begin by clicking on your lab in Author Workbench to view its details page.
```

File: docs/labs/jupyterlab-filelibrary.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+2. **Locate the Content Tab:** In the lab, find and click on the "Content" Tab. Refer to the image below for guidance:
```
Comment (ElioDiNino) [inline docs/labs/jupyterlab-filelibrary.md:7]:
```suggestion
2. **Locate the Content Tab:** In the lab, find and click on the "Content" Tab. Refer to the image below for guidance.
```

File: docs/labs/upgrade-jupyterlab.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+
+# Upgrading to the Latest Version of JupyterLab
+
+Follow these steps to upgrade your JupyterLab Classic to JupyterLab Current:
```
Comment (ElioDiNino) [inline docs/labs/upgrade-jupyterlab.md:4]:
```suggestion
Follow these steps to upgrade your JupyterLab Classic lab to JupyterLab Current:
```

File: docs/labs/upgrade-jupyterlab.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+1. **Open Your Lab:** Begin by launching your lab detail page.
```
Comment (ElioDiNino) [inline docs/labs/upgrade-jupyterlab.md:6]:
```suggestion
1. **Open Your Lab:** Begin by clicking on your lab in Author Workbench to view its details page.
```

File: docs/labs/upgrade-jupyterlab.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+3. **Go to Settings:** Click on the "Settings" option within the Version Tab area.
+![Setting](/img/labs/lab-settings.png)
+
+4. **Choose JupyterLab Version:** Find the section where you can select the lab version. Choose "JupyterLab Current" as shown in the image:
+![Choose Lab Version](/img/labs/choose-newtool.png)
+
+5. **Update Lab Version:** Click on the "Update Lab Version" button to apply the upgrade to the JupyterLab Current.
+
+By completing these steps, your JupyterLab Classic will be successfully updated to the JupyterLab Current.
```
Comment (ElioDiNino) [inline docs/labs/upgrade-jupyterlab.md:19]:
```suggestion
By completing these steps, your JupyterLab Classic lab will be successfully updated to JupyterLab Current! Please make sure to test your lab after updating and make any necessary adjustments.
```

File: docs/skills-network-file-library/how-to-access.md [hunk 7]
```diff
@@ -6,10 +6,10 @@ sidebar_position: 3
 
 Skills Network File Library can be accessed from multiple sources:
 
-- Directly from Author Workbench on the main page of your Guided Project:
+- Directly from Author Workbench on the main page of your Guided Project under `content` tab:
```
Comment (ElioDiNino) [inline docs/skills-network-file-library/how-to-access.md:9]:
```suggestion
- Directly from Author Workbench on the main page of your Guided Project under the "Content" tab
```

File: docs/skills-network-file-library/how-to-access.md [hunk 8]
```diff
@@ -6,10 +6,10 @@ sidebar_position: 3
...
 
-- Directly from Author Workbench on the main page of your Course:
+- Directly from Author Workbench on the main page of your Course under `content` tab:
```
Comment (ElioDiNino) [inline docs/skills-network-file-library/how-to-access.md:11]:
```suggestion
- Directly from Author Workbench on the main page of your Course under the "Content" tab
```

File: docs/skills-network-file-library/how-to-access.md [hunk 9]
```diff
@@ -6,10 +6,10 @@ sidebar_position: 3
...
 
 - Directly from Author IDE when editing your markdown-based instructions:
```
Comment (ElioDiNino) [inline docs/skills-network-file-library/how-to-access.md:13]:
```suggestion
- Directly from Author IDE when editing your markdown-based instructions
```

File: docs/skills-network-file-library/how-to-access.md [hunk 10]
```diff
@@ -6,10 +6,10 @@ sidebar_position: 3
...
 
-- Directly from JupyterLab or JupyterLite when editing your notebooks:
+- Directly from JupyterLab Current or JupyterLite when editing your notebooks:
```
Comment (ElioDiNino) [inline docs/skills-network-file-library/how-to-access.md:15]:
```suggestion
- Directly from JupyterLab Current or JupyterLite when editing your notebooks
```

**Review by ElioDiNino — CHANGES_REQUESTED** [review]

**Review by ElioDiNino — APPROVED** [review]
Looks great!

**Review by bsteinfeld — APPROVED** [review]

**Pull request merged**